src/__experimental__/components/radio-button/radio-button-group.style.js

```diff
@@ -2,6 +2,7 @@ import styled from "styled-components";
 
 const RadioButtonGroupStyle = styled.div`
   ${({ inline }) => inline && "display: flex;"}
+  ${({ legendInline }) => legendInline && "margin-top: 4px;"}
 
   ${({ styleOverride }) => styleOverride};
```

Comment:
I suggest to list all props at the beginning 
```js
const RadioButtonGroupStyle = styled.div`
${({inline, legendInline, styleOverride}) => css``
```

Reply:
good point, changed